libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -240,6 +243,8 @@ public void run() {
             }, t.getDuration());
             t.show();
         }
+        final Intent intent = new Intent(AUTHENTICATION_FAILED_INTENT);
```

Comment:
Also, I was looking at UserAccountManager.USER_SWITCH_INTENT_ACTION but this seems to only be sent if the user is switching to another user that has logged in previously, are there no intents sent for users who have successfully completed the login flow?